pyproject.toml

```diff
@@ -20,7 +20,7 @@ keywords = [
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0.0"
-newrelic = { git = "https://github.com/newrelic/newrelic-python-agent.git", rev = "8e17a97c8e4e869a8af200d65727a70b0a10d568" }
+newrelic = { git = "https://github.com/newrelic/newrelic-python-agent.git", rev = "d74a2d7ef0914cb5ae97028b00f65b093e1a9d73" }
```

Comment:
This commit is the agent with full OpenAI instrumentation from [this branch](https://github.com/newrelic/newrelic-python-agent/tree/develop-ai-limited-preview)